mbs-s0015-20240504/README.md

```diff
@@ -0,0 +1,174 @@
+* The third axis is Stokes component (I, Q, U)
+* The last axes are the map pixels
+
+The geometry (including shape) of the maps is given in the FITS header:
+```
+SIMPLE  =                    T / conforms to FITS standard                      
+BITPIX  =                  -32 / array data type                                
+NAXIS   =
+ 5 / number of array dimensions                     
+NAXIS1  =                10800                                                  
+NAXIS2  =                 2640
+NAXIS3  =                    3                                                  
+NAXIS4  =                    1                                                  
+NAXIS5  =                    2                                                  
+WCSAXES =                    2 / Number of coordinate axes                      
+CRPIX1  =               5401.0 / Pixel coordinate of reference point            
+CRPIX2  =               1890.5 / Pixel coordinate of reference point            
+CDELT1  =   -0.033333333333333 / [deg] Coordinate increment at reference point  
+CDELT2  =    0.033333333333333 / [deg] Coordinate increment at reference point  
+CUNIT1  = 'deg'                / Units of coordinate increment and value        
+CUNIT2  = 'deg'                / Units of coordinate increment and value        
+CTYPE1  = 'RA---CAR'           / Right ascension, plate caree projection        
+CTYPE2  = 'DEC--CAR'           / Declination, plate caree projection            
+CRVAL1  =                180.0 / [deg] Coordinate value at reference point      
+CRVAL2  =                  0.0 / [deg] Coordinate value at reference point      
+LONPOLE =                  0.0 / [deg] Native longitude of celestial pole       
+LATPOLE =                 90.0 / [deg] Native latitude of celestial pole        
+MJDREF  =                  0.0 / [d] MJD of fiducial time                       
+RADESYS = 'ICRS'               / Equatorial coordinate system                   END
+```
+
+## Example usage
+
+We give some minimum working examples:
+```python
+import numpy as np
+from pixell import enmap, curvedsky, enplot
+from os.path import join
+
+# first get some basic info like the path and the filename template
+fdir = '/global/cfs/cdirs/sobs/v4_sims/mbs/mbs_s0015_20240504/sims'
+fbase_template = 'so_lat_mbs_mss0002_fdw_{bands}_lmax5400_4way_set{split_num}_noise_sim_map{sim_num:04}.fits'
+
+# let's load the sim for e.g. f150, split 2, sim 123
+bands = 'mf_f090_mf_f150'
+band_idx = 1                # f090 is 0, f150 is 1
+split_num = 2
+sim_num = 123               # has 4 digits (leading 0's), but padding done for us in template
+
+sim = enmap.read_map(
+    join(fdir, fbase_template.format(
+        bands=bands,
+        split_num=split_num,
+        sim_num=sim_num)
+        ),
+    sel=np.s_[band_idx, 0]  # just load f150, remove split axis
+    )
+print(sim.geometry, sim.dtype)
```

Comment:
I tried to run this code, I get:

    FileNotFoundError: [Errno 2] No such file or directory: '/global/cfs/cdirs/sobs/v4_sims/mbs/mbs_s0015_20240504/sims/so_lat_mbs_mss0002_fdw_mf_f090_mf_f150_lmax5400_4way_set2_noise_sim_map0123.fits'

Please instead of providing so many lines of code inside the README, put it in a separate Python file then reference it from the README file.
This way the file is really easy to test if it is still working, even better if you add a couple of assert statements

Reply:
or a notebook

Reply:
Thanks for checking! I fixed this and moved the examples to separate .py files. The `example_usage.py` file now includes some assert statements like you suggested